### PR TITLE
bun bun!

### PR DESCRIPTION
**Pull request by killer7luis**
## About The Pull Request

This adds a 1% chance of a bunnygirl sprite appearing instead of the training bunny dummy

## Why It's Good For The Game

Its cute :>

## Changelog
:cl:
add: Bunny training dummy girl
:cl:

### CODE REVIEW

**Review by monochromatism — APPROVED** [review]
Yes.

**Review by dsmith328 — COMMENTED** [review]
While I like the art, I'm uncertain about how well it would fit into the game as a whole. Be it pose or just overall. I'm asking @EgorDinamit to review this.

**Comment by killer7luis** [comment]
Btw if you dont like that ver i can change the sprite 
Bungal but not booba preview
into this

**Comment by killer7luis** [comment]
Bungalemergencypreview
Here's also this version just in case

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
I will also not be the one to approve this in code, but I approve it in my soul.
Also change the pixel_x to 32 on sprite change

**Comment by killer7luis** [comment]
How do i do that?

**Review by Kitsunemitsu — COMMENTED** [review]
I'm not going to be the one to approve this in code, but I approve it in my heart.

**Comment by killer7luis** [comment]
The changes chu asked are done

File: code/modules/mob/living/simple_animal/abnormality/teth/training_rabbit.dm [hunk 1]
```diff
@@ -39,3 +43,10 @@
 /mob/living/simple_animal/hostile/abnormality/training_rabbit/proc/kill_dummy()
 	QDEL_NULL(src)
 
+/mob/living/simple_animal/hostile/abnormality/training_rabbit/Initialize()	//1 in 100 chance for bunny girl waifu
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/teth/training_rabbit.dm:46]: Move Initialize proc directly below mob's definition.

**Comment by killer7luis** [comment]
changes doneee

**Review by EgorDinamit — APPROVED** [review]

**Pull request merged**